std/traits.d

```diff
@@ -164,7 +164,7 @@
  */
 module std.traits;
 
-import std.typetuple; // TypeTuple
```

Comment:
It seems like Jenkins fails, because this somehow publicly imported functions like `staticMap`, which is now not the case.